solidity/contracts/maintainer/MaintainerProxy.sol

```diff
@@ -0,0 +1,588 @@
+        _submitDepositSweepProofGasOffset = 26750;
+        _submitRedemptionProofGasOffset = 9750;
+        _submitMovingFundsCommitmentGasOffset = 8000;
+        _resetMovingFundsTimeoutGasOffset = 1000;
+        _submitMovingFundsProofGasOffset = 15000;
+        _notifyMovingFundsBelowDustGasOffset = 3500;
+        _submitMovedFundsSweepProofGasOffset = 22000;
+        _requestNewWalletGasOffset = 3000;
+        _notifyCloseableWalletGasOffset = 4000;
+        _notifyWalletClosingPeriodElapsedGasOffset = 3000;
+        _defeatFraudChallengeGasOffset = 10000;
+        _defeatFraudChallengeWithHeartbeatGasOffset = 5000;
+    }
+
+    /// @notice Wraps submit sweep proof call and reimburses a caller's
+    ///         transaction cost.
+    /// @param sweepTx Bitcoin sweep transaction data
+    /// @param sweepProof Bitcoin sweep proof data
+    /// @param mainUtxo Data of the wallet's main UTXO, as currently known on
+    ///        the Ethereum chain. If no main UTXO exists for the given wallet,
+    ///        this parameter is ignored
+    function submitDepositSweepProof(
+        BitcoinTx.Info calldata sweepTx,
+        BitcoinTx.Proof calldata sweepProof,
+        BitcoinTx.UTXO calldata mainUtxo,
+        address vault
+    ) external onlyMaintainer {
+        uint256 gasStart = gasleft();
+
+        bridge.submitDepositSweepProof(sweepTx, sweepProof, mainUtxo, vault);
+
+        reimbursementPool.refund(
+            (gasStart - gasleft()) + _submitDepositSweepProofGasOffset,
+            msg.sender
+        );
+    }
+
+    /// @notice Wraps submit redemption proof call and reimburses a caller's
+    ///         transaction cost.
+    /// @param redemptionTx Bitcoin redemption transaction data
+    /// @param redemptionProof Bitcoin redemption proof data
+    /// @param mainUtxo Data of the wallet's main UTXO, as currently known on
+    ///        the Ethereum chain
+    /// @param walletPubKeyHash 20-byte public key hash (computed using Bitcoin
+    ///        HASH160 over the compressed ECDSA public key) of the wallet which
+    ///        performed the redemption transaction
+    function submitRedemptionProof(
+        BitcoinTx.Info calldata redemptionTx,
+        BitcoinTx.Proof calldata redemptionProof,
+        BitcoinTx.UTXO calldata mainUtxo,
+        bytes20 walletPubKeyHash
+    ) external onlyMaintainer {
+        uint256 gasStart = gasleft();
+
+        bridge.submitRedemptionProof(
+            redemptionTx,
+            redemptionProof,
+            mainUtxo,
+            walletPubKeyHash
+        );
+
+        reimbursementPool.refund(
+            (gasStart - gasleft()) + _submitRedemptionProofGasOffset,
+            msg.sender
+        );
+    }
+
+    /// @notice Submits the moving funds target wallets commitment.
+    ///         Once all requirements are met, that function registers the
+    ///         target wallets commitment and opens the way for moving funds
+    ///         proof submission.
```

Comment:
The description should be just like in another functions from this file:
```
    /// @notice Wraps submit moving funds commitment and reimburses a caller's
    ///         transaction cost.

Reply:
[e3c6fa0](https://github.com/keep-network/tbtc-v2/pull/209/commits/e3c6fa09610d53a5e77cd695ae416f47cf20cb8f)